ui/src/components/TaskDetailsCard/index.jsx

```diff
@@ -55,6 +55,12 @@ import Link from '../../utils/Link';
   unorderedList: {
     ...theme.mixins.unorderedList,
   },
+  listItemText: {
+    paddingLeft: theme.spacing(2),
+  },
+  codeText: {
+    fontSize: '0.8125rem',
+  },
```

Comment:
Let's remove `codeText`. It's okay if the font is a little bit larger here :)

Reply:
Done!